bin/yoke

```diff
@@ -1,65 +1,19 @@
-#!/bin/env python3
```

Comment:
Actually, isn't the recommended way `/usr/bin/env python3` (so user can have Python installation in their home directory, without admin rights)?

Reply:
You appear to be right: it's the [first allowed shebang](https://docs.python.org/3/using/windows.html#shebang-lines) in Python documentation, and the preferred answer for [this StackOverflow question](https://stackoverflow.com/questions/22222473/shebang-doesnt-work-with-python3).

`/usr/bin/python3` is just the path for Python3 in my working system, as `/bin/env python3` didn't work for me. I just checked and `/usr/bin/env python3` runs the Python3 interpreter in my older computer, so please change it.